### PR TITLE
Improve sanity check for parameter importance

### PR DESCRIPTION
**Pull request by not522**
## Motivation
Follow-up for #3785. This PR divides the sanity check of importance tests by evaluator.

## Description of the changes
- Divide the sanity check by evaluator.

### CODE REVIEW

File: tests/importance_tests/test_init.py [hunk 1]
```diff
@@ -29,6 +31,20 @@
...
 
 
+def _check_param_importance(
+    evaluator: BaseImportanceEvaluator, param_importance: Dict[str, float]
+) -> bool:
+    if len(param_importance) == 0:
+        return True
+
+    if isinstance(evaluator, (MeanDecreaseImpurityImportanceEvaluator, FanovaImportanceEvaluator)):
+        return math.isclose(1.0, sum(i for i in param_importance.values()), abs_tol=1e-5)
```
Comment (contramundum53) [inline tests/importance_tests/test_init.py:41]: I'm not sure if the sum of the importance values should be checked.

**Comment by contramundum53** [comment]
@HideakiImamura Could you review this PR?

**Review by HideakiImamura — APPROVED** [review]
Looks solid. LGTM.

**Comment by not522** [comment]
Let me close this PR because #3828  takes over this issue.

**Pull request closed without merging**